dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/client/LibertyWebServiceClientInInterceptor.java

```diff
@@ -140,21 +153,45 @@ public void handleMessage(Message message) throws Fault {
 
         // Set ignoreUnexpectedElements if true
         if (ignoreUnexpectedElements != null && (boolean) ignoreUnexpectedElements == true) {
-            // Enable validation handling in CXF
-            message.put(JAXBDataBinding.SET_VALIDATION_EVENT_HANDLER, (boolean) ignoreUnexpectedElements);
             
+                // Enable validation handling in CXF
```

Comment:
Formatting needs fixing, its in a few spaces

Reply:
I matched the format of the commits before for readability.